### PR TITLE
[TR] PIM-5612: Add Excel connector supported versions

### PR DESCRIPTION
**Pull request by SamirBoulil**
Adds a connector section into the import/export chapter related to the Microsoft Office supported versions for xlsx files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ Add all Acme bundles in `app/AppKernel.php` file.
 
 ## Contribution
 
-Don't hesitate to propose cookbook entries via https://github.com/akeneo/pim-docs/issues
+Don't hesitate to submit cookbook ideas via https://github.com/akeneo/pim-docs/issues
```
Comment (SamirBoulil) [inline README.md]: Simple suggestion

File: reference/import_export/connectors.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
+XLSX connector
```
Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: Is the filename smart ? I figured we would add all future connector description to this file.

Comment (juliensnz) [inline reference/import_export/connectors.rst]: You should name it akeneo native connector and put a section about csv

File: README.md (hunk 1 above)
Comment (rybus) [inline README.md]: to suggest cookbook ideas ?

**Comment by willy-ahva** [comment]
GTM

File: reference/import_export/connectors.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Akeneo Native Connector
+=================
+
+CSV Format
+-------------
+
+Akeneo Native Connector supports reading and writing CSV files. For more information, please refer to the chapters on product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`) in order to understand how to customize the connector.
```
Comment (rybus) [inline reference/import_export/connectors.rst]: to chapters about product import and product export

File: README.md (hunk 1 above)
Comment (SamirBoulil) [inline README.md]: Yes right! even better !

File: reference/import_export/connectors.rst (hunk 3 above)
Comment (rybus) [inline reference/import_export/connectors.rst]: You should put the end of your sentence at its beginning: "In order to customize the connector, you can refer to the chapters about product import and export."

Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: thanks ;)

**Comment by solivier** [comment]
gtm

File: reference/import_export/connectors.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Akeneo Native Connector
+=================
```
Comment (nidup) [inline reference/import_export/connectors.rst]: formatting issue here? you should have same number of "=" than the title contains characters

Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: Oupsiz yes right !

File: reference/import_export/connectors.rst [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+CSV Format
+-------------
+
+Akeneo Native Connector supports reading and writing CSV files. In order to understand how to customize the connector, please refer to chapters about product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`).
+
+XLSX Format
+--------------
```
Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: Same here !

File: reference/import_export/connectors.rst [hunk 6]
```diff
@@ -0,0 +1,21 @@
+Akeneo Native Connector
+=================
+
+CSV Format
+-------------
+
+Akeneo Native Connector supports reading and writing CSV files. In order to understand how to customize the connector, please refer to chapters about product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`).
```
Comment (nidup) [inline reference/import_export/connectors.rst]: imho,
"how to customize" is more related to cookbooks
"understand how it works" is related to reference doc

this connector allow to import/export numerous PIM objects not only products

**Comment by nidup** [comment]
We discussed with @Delphineray about splitting the native connector in a CSV connector and an Excel connector. We'll change this doc when it will be done. Baby comments and GTM.

BTW @pierallard is working on a PR to explain how simple imports (other objects than product) works

File: reference/import_export/connectors.rst [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+XLSX Format
+--------------
+
+The connector uses an open source library called Spout (https://github.com/box/spout) to read and create XLSX files.
+
+The connector has been tested and works with the following Microsoft Office versions:
```
Comment (phaseinducer) [inline reference/import_export/connectors.rst]: I'd say "The connector supports the following Excel versions:"

File: reference/import_export/connectors.rst [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+- Windows Excel 2007
+- Excel 16 (Office 2016 for OS X)
+- Excel 14 (Office 2011 for OS X)
+
+Customizing connectors
+----------------------
+
+It is possible to customize this connector to import/export numerous PIM related objects:
+
+- How to import PIM objects using this connector (cf :doc:`reference/import_export/simple-import.rst`)
```
Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: @nidup Is it okay if I add it now in my PR though @pierallard's PR is not merge yet ?

File: reference/import_export/connectors.rst (hunk 7 above)
Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: Yeah clearer/shorter.

**Pull request merged**